src/common/util.cc

```diff
@@ -145,6 +167,7 @@ static void distro_detect(map<string, string> *m, CephContext *cct)
 
 int get_cgroup_memory_limit(uint64_t *limit)
 {
+  #ifndef _WIN32
```

Comment:
i'd prefer avoid indent unless necessary, for instance it's nested in yet another `#ifdef`.